concert/app/src/layer/python/enchanted_brain/attributes.py

```diff
@@ -1,5 +1,6 @@
 ATTR_RECORD_ID = "recordId"
 ATTR_CREATED_AT = "createdAt"
+ATTR_TOTAL_CHOICES = "choices"
```

Comment:
Please call this `ATTR_AGGREGATE_TOTAL_CHOICES` to be consistent with record-type-specific attributes.

Reply:
Done.